--- a/qtbase/mkspecs/common/clang.conf
+++ b/qtbase/mkspecs/common/clang.conf
@@ -4,8 +4,8 @@
 
 QMAKE_COMPILER          = gcc clang llvm   # clang pretends to be gcc
 
-QMAKE_CC                = $${CROSS_COMPILE}clang
-QMAKE_CXX               = $${CROSS_COMPILE}clang++
+QMAKE_CC                = clang
+QMAKE_CXX               = clang++
 
 QMAKE_LINK_C            = $$QMAKE_CC
 QMAKE_LINK_C_SHLIB      = $$QMAKE_CC
